tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Requests/GetOrderItemRequest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Responses;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Utils;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Requests
+{
+    internal sealed class GetOrderItemRequest
+    {
+        private readonly Request _request;
+        private readonly string _getOrderItemUrl;
+
+        public GetOrderItemRequest(
+            Request request,
+            string orderingApiBaseAddress,
+            string orderId,
+            int? orderItemId)
```

Comment:
Can the orderItemId be of type int? 
In the controller its passed in as type int (unless I'm missing something)

Reply:
Yeah I was attempting to use this as a "not found" test. In the end I went with another step, which still not sure of but yeah will put this back to an int.